Assets/HoloToolkit/Utilities/Shaders/FastConfigurable.shader.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: a82e601d90c076c458969fbd127780e6
+guid: f1f3948b20c230044bda31b620ddd37f
```

Comment:
Haha the probability is low, but sometimes can happen. Just gotta use really big prime numbers when generating!